lua/spectre_common.lua

```diff
@@ -23,6 +23,12 @@ local HEADERS = {
     ZIPKIN_ID = 'X-Zipkin-Id',
 }
 
+local SUPPORTED_POST_ENCODING = {
```

Comment:
Can you unify this with `lua/bulk_endpoints.lua`? We also enforce that bulk endpoints encode their responses with "application/json". So this constant should be:

* renamed to something more generic like "SUPPORTED_ENCODING_FOR_ID_EXTRACTION"
* used both here and in `lua/bulk_endpoints.lua`